string command
float32 numarg
